config.yml

```
@@ -302,6 +302,8 @@ params:
   customTabs:
     - name: UptimeRobot
       link: https://stats.uptimerobot.com/7nArVImJnG
+    - name: Testing Farm Status
+      link: https://testing-farm.gitlab.io/status/
```

```suggestion
https://status.testing-farm.io
```

This would be better, we might move from gitlab sometime ...